src/app.module.ts

```diff
@@ -54,6 +55,34 @@ import { MembershipModule } from '@src/services/domain/membership/membership.mod
           ?.logging,
       }),
     }),
+    TypeOrmModule.forRootAsync({
+      name: 'jolocom',
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: async (configService: ConfigService) => ({
+        type: 'sqlite', //todo: switch to mysql when issue is addressed.
+        insecureAuth: true,
+        synchronize: true /* note: only for demo */,
+        cache: true,
+        entities: [
+          'node_modules/@jolocom/sdk-storage-typeorm/js/src/entities/*.js',
+        ],
+        // host: configService.get(ConfigurationTypes.Identity)?.ssi.jolocom.database
```

Comment:
why do you keep all those commented lines?

Reply:
am waiting for a fix from jolocom; once that is in then want to enable mysql storage.